capstone/src/main/java/com/google/sps/servlets/Student.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.servlets;
+
+import com.google.common.collect.ImmutableList;
+
+/** Each instance of Student contains all necessary student information */
+public class Student {
+  private String name;
+  private Integer gradYear;
+  private String major;
+  private String email;
+  private ImmutableList<String> clubs;
```

Comment:
For the Student class, I feel like this should be a normal list because students can join and leave clubs

Reply:
I agree, I was thinking of using ImmutableList for now and changing to List when the leave function works, but changing it now should be fine as well :)